index.d.ts

```diff
@@ -18,7 +18,7 @@ declare namespace unless {
          * and methods key-pairs. If the request path or path and method match, the middleware will not run. Check
          * Examples for usage.
          */
-        path?: string | string[];
+        path?: string | RegExp | (string | RegExp | { url: (string | RegExp); methods: string[]; })[];
```

Comment:
Could you link to the docs/implementation that say that
- a regexp is supported for `url`
- you can mix string, regexp and objects in the array (the comment says you cant)